libs/cardano-ledger-core/src/Cardano/Ledger/State/StakePool.hs

```diff
@@ -438,35 +437,35 @@ instance DecCBOR PoolMetadata where
   decCBOR = do
     decodeRecordNamed "PoolMetadata" (const 2) (PoolMetadata <$> decCBOR <*> decCBOR)
 
--- | The size of the 'ppOwners' 'Set'.  Only used to compute size of encoded
--- 'PoolParams'.
+-- | The size of the 'sppOwners' 'Set'.  Only used to compute size of encoded
+-- 'StakePoolParams'.
 data SizeOfPoolOwners = SizeOfPoolOwners
 
 instance EncCBOR SizeOfPoolOwners where
   encCBOR = error "The `SizeOfPoolOwners` type cannot be encoded!"
 
--- | The size of the 'ppRelays' 'Set'.  Only used to compute size of encoded
--- 'PoolParams'.
+-- | The size of the 'sppRelays' 'Set'.  Only used to compute size of encoded
+-- 'StakePoolParams'.
 data SizeOfPoolRelays = SizeOfPoolRelays
 
 instance EncCBOR SizeOfPoolRelays where
   encCBOR = error "The `SizeOfPoolRelays` type cannot be encoded!"
 
-instance EncCBORGroup PoolParams where
-  encCBORGroup poolParams =
-    encCBOR (ppId poolParams)
-      <> encCBOR (ppVrf poolParams)
-      <> encCBOR (ppPledge poolParams)
-      <> encCBOR (ppCost poolParams)
-      <> encCBOR (ppMargin poolParams)
-      <> encCBOR (ppRewardAccount poolParams)
-      <> encCBOR (ppOwners poolParams)
-      <> encCBOR (ppRelays poolParams)
-      <> encodeNullMaybe encCBOR (strictMaybeToMaybe (ppMetadata poolParams))
+instance EncCBORGroup StakePoolParams where
+  encCBORGroup stakePoolParams =
+    encCBOR (sppId stakePoolParams)
+      <> encCBOR (sppVrf stakePoolParams)
+      <> encCBOR (sppPledge stakePoolParams)
+      <> encCBOR (sppCost stakePoolParams)
+      <> encCBOR (sppMargin stakePoolParams)
+      <> encCBOR (sppRewardAccount stakePoolParams)
+      <> encCBOR (sppOwners stakePoolParams)
+      <> encCBOR (sppRelays stakePoolParams)
+      <> encodeNullMaybe encCBOR (strictMaybeToMaybe (sppMetadata stakePoolParams))
```

Comment:
I know this is not part of the rename, but since this is being changed, might as well adjust it:
```suggestion
      <> encodeNullStrictMaybe encCBOR (sppMetadata stakePoolParams)
```